src/main/scala/com/zeotap/data/io/common/types/SupportedFeatures.scala

```diff
@@ -54,6 +54,9 @@ object SupportedFeatures {
 
   final case class LatestPaths[A](pathTemplate: String, parameters: Map[String, String], relativeToCurrentDate: Boolean) extends SupportedFeatures[A]
 
+  //Dataframe-specific features
```

Comment:
inappropriate comment here. Comment should mention why we came up with this feature especially the single file issue and how it can be relevant in any scenario like apache Beam and not just spark